pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -31,22 +29,22 @@ in {
       url = "https://github.com/vhyza/elasticsearch-analysis-lemmagen/releases/download/v${version}/${name}-plugin.zip";
       sha256 = "1m4z05wixjrq4nlbdjyhvprkrwxjym8aba18scmzfn25fhbjgvkz";
     };
-    meta = {
+    meta = with pkgs.lib; {
```

Comment:
Isn't `stdenv.lib` normally used? I don't really know the difference, but that seems to be the convention.

Reply:
fixed